### PR TITLE
Documentation update

### LINKED ISSUE #241: Documentation  Update
A review of the current `bounty-bot` documentation is needed.

The documentation for this project appears to be outdated. It does not reflect the current state of the software and may cause confusion for users.

To address this issue, I recommend updating the documentation to reflect any changes that have been made to the project. This may involve rewriting certain sections, adding new content, or removing outdated information.

### PR DESCRIPTION
**Pull request by devpanther**
Fixes #241

### CODE REVIEW

**Comment by devpanther** [comment]
Screenshot 2023-04-11 at 10 25 10 AM
This alllowed? @pavlovcik

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+```
+ _____ _              _   _ _     _             _______       _   
+|_   _| |            | | | | |   (_)           (_) ___ \     | |  
+  | | | |__   ___    | | | | |__  _  __ _ _   _ _| |_/ / ___ | |_ 
+  | | | '_ \ / _ \   | | | | '_ \| |/ _` | | | | | ___ \/ _ \| __|
+  | | | | | |  __/   | |_| | |_) | | (_| | |_| | | |_/ / (_) | |_ 
+  |_| |_| |_|\___|    \___/|_.__/|_|\__, |\__,_|_\____/ \___/ \__|
+                                     | |                        
+                                     |_|                                              
+```
```
Comment (pavlovcik) [inline README.md]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "build": "tsc",
     "build:serverless": "ncc build src/index.ts -o dist",
     "start": "probot run ./lib/index.js",
-    "start:watch": "node copy.js && src/assets/images lib/assets/images && nodemon --exec 'yarn start'",
+    "start:watch": "node copy.js && nodemon --exec 'yarn start'",
```
Comment (pavlovcik) [inline package.json:17]: Why was this changed?

File: app.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ default_events:
   # - deployment_status
   # - fork
   # - gollum
-  # - issue_comment
+  - issue_comment
```
Comment (pavlovcik) [inline app.yml:25]: Why was this modified?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,15 @@
...
+```js
+ _____ _              _   _ _     _             _______       _   
+|_   _| |            | | | | |   (_)           (_) ___ \     | |  
+  | | | |__   ___    | | | | |__  _  __ _ _   _ _| |_/ / ___ | |_ 
+  | | | '_ \ / _ \   | | | | '_ \| |/ _` | | | | | ___ \/ _ \| __|
+  | | | | | |  __/   | |_| | |_) | | (_| | |_| | | |_/ / (_) | |_ 
+  |_| |_| |_|\___|    \___/|_.__/|_|\__, |\__,_|_\____/ \___/ \__|
+                                     | |                        
+                                     |_|                                              
+```
```
Comment (pavlovcik) [inline README.md]:
```suggestion
```

I'm not sure that this will render correctly on the README. It's better to make a real image some day in the future.

**Review by pavlovcik — CHANGES_REQUESTED** [review]
I expect that no code should be changed in the documentation update bounty. Please explain if you want to keep those changes.

**Comment by pavlovcik** [comment]
Hey @0xcodercrane do these instructions look correct?

@Steveantor I would appreciate if you could also verify!

File: README.md (hunk 4 above)
Comment (devpanther) [reply README.md]: > I'm not sure that this will render correctly on the README. It's better to make a real image some day in the future.

It renders correctly though, i tested it out and it works when i put it in a code block

File: package.json (hunk 2 above)
Comment (devpanther) [reply package.json:17]: `copy.js` already copies the images over and besides the second line breaks on mac so its best to remove

Comment (Steveantor) [reply package.json:17]: not sure who added that line in the first place and why

File: app.yml (hunk 3 above)
Comment (Steveantor) [reply app.yml:25]: could u tell how the bot is still responding to commands like `/help`

Comment (devpanther) [reply app.yml:25]: > could u tell how the bot is still responding to commands like `/help`

Just saying it doesn't work for me currently without that, you can setup a fresh installation and see

I think GitHub separated the issue comments from the issue events like tags so i had to enable that too

Comment (0xcodercrane) [reply app.yml:25]: yes it's used by default when initially configuring the app. the reason why ubiquibot is still responding is that we've updated app settings manually after the first configuration. agreed with @devpanther 's change.

**Review by 0xcodercrane — CHANGES_REQUESTED** [review]
The ubiquibot is no longer used as a github action. lets remove related stuff from readme.

**Comment by 0xcodercrane** [comment]
> Hey @0xcodercrane do these instructions look correct?
there are something missing. requested changes!

File: .env.example [hunk 5]
```diff
@@ -1,5 +1,6 @@
 # Use `trace` to get verbose logging or `info` to show less
 LOG_LEVEL=debug
+LOGDNA_INGESTION_KEY=
```
[the remark this answers is not in the document]
Comment (devpanther) [reply .env.example:3]: Doesn't need `PERMIT_BASE_URL` and `CHAIN_ID` anymore? @0xcodercrane

File: README.md [hunk 6]
```diff
@@ -82,13 +114,15 @@ Bounty bot is built using the [probot](https://probot.github.io/) framework so i
 You can use the bounty bot in 2 ways: [github action](https://github.com/marketplace/actions/ubiquity-bounty-bot) or [github app](https://github.com/marketplace/ubiquibot).
```
[the remark this answers is not in the document]
Comment (devpanther) [reply README.md:114]: > Right now, the bounty bot has been updated as a github app, no longer used as a github action. We should update everything that is related to github action.

The `How to use` should be removed as well right? or maybe replace with markeplace link to install the bot

**Comment by pavlovcik** [comment]
> The ubiquibot is no longer used as a github action. lets remove related stuff from readme.

I am lamenting removing this because I still think that for bounty hunters to work on the bot, and for us to be able to most conveniently test changes, it should have a "dev mode" that operates entirely off of GitHub actions as we used to have it, but from each person's repository as I have continued to mention to you @0xcodercrane.

**Comment by devpanther** [comment]
@pavlovcik @0xcodercrane requested changes implemented

**Comment by devpanther** [comment]
> > The ubiquibot is no longer used as a github action. lets remove related stuff from readme.
> 
> it should have a "dev mode" that operates entirely off of GitHub actions as we used to have it

Wondering why it was removed, everyone can setup a dev version of it though

**Comment by 0xcodercrane** [comment]
To setup it on GitHub action for testing purpose, I might have to update the codebase as much as I did (from GitHub action to GitHub app). this means we need to keep the different codebases in the source even though its not a lot. 

But if you guys still think, its mandatory to keep an action-based bot, I will add it to the todo list.

**Comment by 0xcodercrane** [comment]
> Wondering why it was removed

Because it had several annoying steps in the distribution. Also, probot-github-action has its own limits in terms of permission/usage.

**Review by 0xcodercrane — APPROVED** [review]

**Comment by devpanther** [comment]
> > Wondering why it was removed
> 
> Because it had several annoying steps in the first distribution. Also, probot-github-action has its own limits in terms of permission/usage.

Oh, makes sense

**Comment by devpanther** [comment]
@pavlovcik can you review?

**Review by pavlovcik — APPROVED** [review]

**Comment by pavlovcik** [comment]
> Also, probot-github-action has its own limits in terms of permission/usage.

For whatever its worth, regarding permissions I remember one issue was purely cosmetic: it was to post comments as "UbiquiBot." We can use the default option, `github-actions` with the staging environments. Perhaps the other permissions are also unnecessary to accommodate for this use case.
